docs/contributing.md

```diff
@@ -0,0 +1,115 @@
+# Contibuting
+
+All contributions to *databases* are welcomed!
+
+## Issues
+
+To make it as simple as possible for us to help you, please include the following:
+
+* OS 
+* python version
+* databases version
+* database backend (mysql, sqlite or postgresql)
+* database driver (aiopg, aiomysql etc.)
+
+Please try to always include the above unless you're unable to install *databases* or **know** it's not relevant
+to your question or feature request.
+
+## Pull Requests
+
+It should be quite straight forward to get started and create a Pull Request.
+
+!!! note
+    Unless your change is trivial (typo, docs tweak etc.), please create an issue to discuss the change before
+    creating a pull request.
+
+To make contributing as easy and fast as possible, you'll want to run tests and linting locally. 
+
+You'll need to have **python >= 3.6 (recommended 3.7+)** and **git** installed.
+
+## Getting started
+
+1. Clone your fork and cd into the repo directory
+```bash
+git clone git@github.com:<your username>/databases.git
+cd databases
+```
+
+2. Create and activate virtual env
+```bash
+virtualenv -p `which python3.6` env
+source env/bin/activate
+```
+
+3. Install databases, dependencies and test dependencies
+```bash
+pip install -r requirements.txt
+```
+
+4. Checkout a new branch and make your changes
+```bash
+git checkout -b my-new-feature-branch
+```
+
+## Make your changes...
+
+## Contribute
+
+1. Formatting and linting - databases uses black for formatting, autoflake for linting and mypy for type hints check
```

Comment:
The `./scripts/lint` won't run mypy, when you run `./scripts/test` it will call `./scripts/check` which does those checks. Needs a bit of re-wording.

Reply:
why? It should run mypy:

https://github.com/encode/databases/blob/f8bff8f4f4197d12dcd7489c15c55a7e5a62ab2e/scripts/lint#L10-L13

Reply:
Should we unify this or do I leave it as it is?

Reply:
I think it's worth unifiying it, but not that important really. Up to you.